planning/behavior_path_planner/src/scene_module/lane_change/normal.cpp

```diff
@@ -694,6 +696,14 @@ LaneChangeTargetObjectIndices NormalLaneChange::filterObject(
       min_dist_ego_to_obj = std::min(dist_ego_to_obj, min_dist_ego_to_obj);
     }
 
+    const auto is_lateral_far = [&]() {
+      const auto dist_object_to_current_lanes_center =
+        lanelet::utils::getLateralDistanceToClosestLanelet(
+          current_lanes, object.kinematics.initial_pose_with_covariance.pose);
+      const auto lateral = dist_object_to_current_lanes_center - dist_ego_to_current_lanes_center;
+      return std::abs(lateral) > (common_parameters.vehicle_width / 2);
```

Comment:
why the judgement is based on the `vehicle_width`?

Is it not sufficient to check if the center of the vehicle is within the target lane?

Reply:
@kyoichi-sugahara 
I'm using the judgement based on vehicle width instead, because
1. the `width` of the lanes might vary.
2. half of the target vehicle might already in target lane (but the center still in current lane)